.gitignore

```diff
@@ -45,6 +45,9 @@ default_env.tres
 # Windows
 *.bat
 
+# OSX
+*.os
```

Comment:
What is this extension? Last time I saw it, it was actually a very old typo, which should have been `.so` or `.o`, referring to object files during the build.

Reply:
Looks like it some kind of object file, these files were created after build:
```
bash-3.2$ ls
gd_library.cpp
gd_library.os
image_utils.cpp
image_utils.os
...

bash-3.2$ file gd_library.os 
gd_library.os: Mach-O 64-bit object x86_64
```

Reply:
I'm prettty sure it's a typo in GDNative's build files. It should have been `.o`

Reply:
Yeah, probably you are right. Probably .gitignore should not be changed then. Unfortunately I cannot find why this files are named this way. But I followed the build instructions :)

Reply:
In this .gitignore *.os is listed in section # Compiled source #
https://github.com/2shady4u/godot-sqlite/blob/master/.gitignore
but probably it is a typo in GDNative's build file too, which was copied across the repositories.